src/k-provider/ovp/loaders/bumper-entry-loader.js

```diff
@@ -0,0 +1,86 @@
+//@flow
+import RequestBuilder from '../../../util/request-builder';
+import OVPBaseEntryService from '../services/base-entry-service';
+import OVPConfiguration from '../config';
+import KalturaPlaybackContext from '../response-types/kaltura-playback-context';
+import KalturaMediaEntry from '../response-types/kaltura-media-entry';
+import KalturaBaseEntryListResponse from '../response-types/kaltura-base-entry-list-response';
+import Error from '../../../util/error/error';
+
+type OVPBumperEntryLoaderResponse = {
+  entry: KalturaMediaEntry,
+  playBackContextResult: KalturaPlaybackContext
+};
+export type {OVPBumperEntryLoaderResponse};
+
+export default class OVPBumperEntryLoader implements ILoader {
+  _entryId: string;
+  _requests: Array<RequestBuilder>;
+  _response: ?OVPBumperEntryLoaderResponse = null;
+
+  static get id(): string {
+    return 'bumper';
+  }
+
+  /**
+   * @constructor
+   * @param {Object} params loader params
+   */
+  constructor(params: Object) {
+    this.requests = this.buildRequests(params);
+    this._entryId = params.entryId;
+  }
+
+  set requests(requests: Array<RequestBuilder>) {
+    this._requests = requests;
+  }
+
+  get requests(): Array<RequestBuilder> {
+    return this._requests;
+  }
+
+  set response(response: any) {
+    if (response[0].hasError) {
+      new Error(Error.Severity.CRITICAL, Error.Category.SERVICE, Error.Code.INVALID_ENTRY_ID, {
+        messages: `OVPBumperEntryLoader - ${response[0].error.message}`
+      });
+    } else {
+      let mediaEntryResponse: KalturaBaseEntryListResponse = new KalturaBaseEntryListResponse(response[0].data);
+      this._response = {
+        entry: mediaEntryResponse.entries[0],
+        playBackContextResult: new KalturaPlaybackContext(response[1].data)
+      };
+    }
+  }
+
+  get response(): ?OVPBumperEntryLoaderResponse {
+    return this._response;
+  }
+
+  /**
+   * Builds loader requests
+   * @function
+   * @param {Object} params Requests parameters
+   * @returns {RequestBuilder} The request builder
+   * @static
+   */
+  buildRequests(params: Object): Array<RequestBuilder> {
+    const config = OVPConfiguration.get();
+    const requests: Array<RequestBuilder> = [];
+    const listRequest = OVPBaseEntryService.list(config.serviceUrl, params.ks, params.entryId, params.redirectFromEntryId);
+    const playbackContextRequest = OVPBaseEntryService.getPlaybackContext(config.serviceUrl, params.ks, params.entryId);
+    listRequest.mandatory = playbackContextRequest.mandatory = false;
```

Comment:
seems like this should get to the provider level and not be added at all if no provider ID was set

Reply:
`mandetory` means this request should not fail the entire multi request if it gets an error from the BE